### PR TITLE
create help guide panel

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12045

*Description of changes:* create help guide panel

image

when cursor hover on top of the panel:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by garyfan1** [comment]
I'll update the version number once the bcgov/bcregistry#74 gets merged.

File: pages/dashboard.vue [hunk 1]
```diff
@@ -31,6 +31,22 @@
               Administrator of your BC Registries account.
             </p>
           </v-container>
+          <a
+          class="text-decoration-none" target="_blank" rel="noopener noreferrer"
+          :href="'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/'
+            +'permits-licences/news-updates/modernization-updates/using-the-business-registry'">
```
Comment (cameron-freshworks) [inline pages/dashboard.vue:37]: Indentation and line arrangement needs to be cleaned up here

File: pages/dashboard.vue [hunk 2]
```diff
@@ -31,6 +31,22 @@
...
+            <v-container fluid rounded class="dash-container-info service-container mt-5 white">
+              <h4>Help with How to Use BC Registries Products and Services</h4>
+              <p class="ma-0 pt-3">
+                Learn about the registries and how to use the applications through step-by-step guides,
+                 online videos, and downloadable quick guides.
```
Comment (cameron-freshworks) [inline pages/dashboard.vue:42]: You have additional spacing here

File: pages/dashboard.vue [hunk 3]
```diff
@@ -31,6 +31,22 @@
...
+              </p>
+              <v-row no-gutters class="align-bottom" style="padding-top:30px;">
```
Comment (cameron-freshworks) [inline pages/dashboard.vue:44]: Use helper classes if you can

File: pages/dashboard.vue [hunk 4]
```diff
@@ -184,4 +200,12 @@ export default Vue.extend ({
   font-size: $px-18;
   padding-top: 50px;
 }
+
+.service-container {
+  cursor: pointer;
+
+  &:hover {
+    box-shadow: 1px 1px 6px 0px $gray6;
+  }
```
Comment (cameron-freshworks) [inline pages/dashboard.vue:209]: Was box shadow called out in the Visual Design specs?

File: pages/dashboard.vue (hunk 3 above)
Comment (garyfan1) [reply pages/dashboard.vue:44]: the spec says 30px, but Vuetify only has pt-7 (28px) or pt-8 (32px), so I ended up with inline styling.

File: pages/dashboard.vue (hunk 4 above)
Comment (garyfan1) [reply pages/dashboard.vue:209]: yes, I think this is what the spec wants.
image

File: pages/dashboard.vue (hunk 3 above)
Comment (cameron-freshworks) [reply pages/dashboard.vue:44]: imo just use the helpers. 2px should be close enough 👍

File: pages/dashboard.vue (hunk 4 above)
Comment (cameron-freshworks) [reply pages/dashboard.vue:209]: Ok cool, thanks for verifying 👍

Comment (cameron-freshworks) [reply pages/dashboard.vue:209]: Resolved (sorry don't have permissions here to do it :P)

File: pages/dashboard.vue (hunk 2 above)
Comment (garyfan1) [reply pages/dashboard.vue:42]: oh I didn't know newline is treated as a space! fixed

File: pages/dashboard.vue (hunk 3 above)
Comment (garyfan1) [reply pages/dashboard.vue]: ok, updated!

**Pull request merged**